### ISSUE TITLE
[question] always show torrent details tab (attribute)

### ISSUE THREAD

**Issue by zilexa** [ronggang/transmission-web-control#477]
* Transmssion version: 3
* Tr Web Control version: 1.6.1
* Browser name & version：Firefox 80.0
* Problem description: How to always show the bottom area (called Attribute) with all the torrent details? Because it is hidden by default and when opened, it automatically hides when the cursor moves away. I don't want it to auto-hide and also want to keep it visible constantly. Is this possible?

![Screenshot from 2020-10-05 12-27-07](https://user-images.githubusercontent.com/3430004/95068926-47290180-0706-11eb-899b-32c4a891bc75.png)

**Comment by imebeh** [comment]
click the right-top arrow to pin or unpin

**Comment by Stephen-King** [comment]
Just click the pattern with the left mouse button.
![FireShot Pro Webpage Screenshot #001 - '95068926-47290180-0706-11eb-899b-32c4a891bc75_png (2640×1216)' - user-images_githubusercontent_com](https://user-images.githubusercontent.com/8936000/95217986-53cf5780-07e3-11eb-86ed-1fead9736c60.png)

**Comment by zilexa** [comment]
That just drops it down again.. it's either automatically dropping (loss of focus) or if I click those arrows it drops again immediately.

**Comment by Stephen-King** [comment]
Click it
![FireShot Pro Webpage Screenshot #002 - 'Transmission WEB 控制器 1_6_1' - http](https://user-images.githubusercontent.com/8936000/95280506-abea7600-0844-11eb-8cce-dc5aece1a84e.png)

**Comment by zilexa** [comment]
I can make a video of my mouse cursor clicking on it if you need proof. 
It's just poorly designed, not to offend the developers. But here is how it currently "works":

Situation: the section is hidden.
User: click on the blue bar.
Result: section is shown.

User: move cursor away. 
Result: section automatically drops down. 

User: Clicks on blue bar again.
Result: section is shown. 

User: now carefully clicks on the arrow pointing down. 
Result: section is hidden. 

Situation: the section is hidden. 
User: click exactly on the arrow. 
Result: section is shown, does not automatically drop down. 

Would you agree this is not intuitive at all? It's puzzling. You now have an arrow pointing up or down that CAN function as a pin, only when the section is hidden and the user clicks on the up arrow. 

The pin should be visible on the blue bar (when the section is shown) and function as.. a pin.

Feel free to close if you disagree and will leave it as is. It could be a minor improvement that does make it easier for most people to understand.

**Comment by Stephen-King** [comment]
The developer may consider that auto-hiding can free up more space and allow us to see more downloads.

**Comment by zilexa** [comment]
My case is not against auto-hiding, that is fine and definitely useful!

My case is to change the well-hidden pinning feature :) 
Hope the dev will read this as I have no other way to reach him.

**Comment by imebeh** [comment]
![sc_20201007_210530](https://user-images.githubusercontent.com/1243962/95334602-edc8fa00-08e0-11eb-9011-fe39b4a0396c.gif)

when auto hide enabled: 
 - show: click anywhere of the toolbar but not the switch button
 - hide: coursor move out of the attribute box

**Comment by MitchTalmadge** [comment]
I agree. This is not intuitive **at all.** Please change the arrow icon to a pin icon. 

Right now if the window is open, clicking the icon does not pin the window. Instead, it lowers the window. If the window is open, clicking the pin icon should keep it open when the mouse cursor is moved away. Clicking it again should make it so that when you move the mouse cursor away, the window drops down again as expected.

**Comment by ronggang** [comment]
This is the native function of `easyui`, maybe we can consider replacing it with a new icon.
